packages/mendel-browserify/index.js

```diff
@@ -7,6 +7,7 @@ var xtend = require('xtend');
 var defined = require('defined');
 var fs = require('fs');
 var path = require('path');
+var isarray = require('isarray');
```

Comment:
Mendel codebase is ES5 safe, the tests will run in NodeJS 0.12 because I want teams at Yahoo that are on 0.12 to use Mendel.